packages/devtools/lib/src/memory/memory.dart

```diff
@@ -667,6 +876,53 @@ class MemoryScreen extends Screen with SetStateMixin {
     }
   }
 
+  bool _matchPattern(String classPattern, String className) {
+    String pattern;
+
+    classPattern = classPattern.trim();
+    if (classPattern.length == 1 && classPattern == '*') return true;
```

Comment:
my preference would be to either support full regexp or no regexp so users don't have to think about what subset of regexp you support. Another option could be to act like there is always a star at the start and end of the pattern and return any classes that start with or end with classPattern screen.

Reply:
Class pattern can be:
```
     ''        - matches everything.
     '*'       - matches everything.
     'NNN*'    - matches a class name starting with NNN.
     '*NNN'    - matches a class name ending with NNN.
     'NNN*MMM' - matches a class name starting with NNN and ending with MMM.
     'NNN'     - matches a class name starting with NNN.
```